transactions/burn-flow/2022/sep-21/README.md

```diff
@@ -1,10 +1,17 @@
 # Burn Flow - Sep 21
 
 The Flow Service account will burn 38,357,187 FLOW after it has been
-transferred from the Genesis Token Account. These tokens are accounted for as
-part of the FlowToken.totalSupplybut are *not currently considered* as part of
+transferred from the Genesis Token Account. These tokens are "Bonus Tokens" and 
+are accounted for as part of the FlowToken.totalSupply but are *not currently considered* as part of
```

Comment:
update to "not considered"
```suggestion
are accounted for as part of the FlowToken.totalSupply but are *not considered* as part of
```

"currently" implies that will change which it won't. These tokens won't exist after they are burnt and there won't again be a notion of bonus tokens.